src/test/java/org/cactoos/collection/NoNullsTest.java

```diff
@@ -120,4 +130,145 @@ void throwsErrorIfNullInAddAll() {
             )
         ).affirm();
     }
+
```

Comment:
@andreoss Please, add a test to check a `new NoNulls(new ListOf<>())` (a list already created empty).

Reply:
@fabriciofx Added